src/middleware/image-upload/multer.ts

```diff
@@ -0,0 +1,39 @@
+import multer from "multer"
+import path from 'path';
+import { RequestHandler} from 'express';
+
+
+// Set storage engine for multer
+const storage = multer.diskStorage({
+    destination: (req, file, cb) => {
+        cb(null, 'uploads/');
```

Comment:
Will this work for Vercel? is there a way to test it? I think we have builds for PR-s, right? let's check if this is working on staging deployment.